app/javascript/components/MentorPageLoadingView.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+const MentorPageLoadingView = ({ rows, columns }) => {
```

Comment:
細かなところが気になってしまいました🙇‍♂️
関数コンポーネントの記述順に合わせて下記はいかがでしょうか？
```suggestion
const MentorPageLoadingView = ({ columns, rows }) => {
```
採用いただける場合、LoadingMentorPageCategiesPlaceholder.jsx の記述順も修正いただければと思います🙇‍♂️

Reply:
> 細かなところが気になってしまいました🙇‍♂️
関数コンポーネントの記述順に合わせて下記はいかがでしょうか？

記述順がごちゃごちゃになっていると読みづらいですね…ご指摘ありがとうございます！
LoadingMentorPageCategiesPlaceholder.jsxと合わせて修正いたしました😊